### PR TITLE
API APP Key documentation

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

* Creates API documentation for the APP and API keys management
* Updates the postman collection with those new endpoints

### Motivation
 It was public and not documented

### Preview link

* https://docs-staging.datadoghq.com/gus/api-api-key/api/?lang=bash#key-management

### CODE REVIEW

File: content/en/api/key_management/api_key.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: API Keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/api_key.md]:
```suggestion
title: API keys
```

File: content/en/api/key_management/api_key.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+type: apicontent
+order: 16.1
+external_redirect: /api/#api-keys
+---
+
+## API Keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/api_key.md]:
```suggestion
## API keys
```

File: content/en/api/key_management/api_key.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Manage all your API Key with the following endpoints:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/api_key.md]:
```suggestion
Manage all your API keys with the following endpoints:
```

File: content/en/api/key_management/api_key.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+* To **get** all API keys available for your account:
+
+    **`GET /api/v1/api_key`**
+
+* To **get** a given API key:
+
+    **`GET /api/v1/api_key/<API_KEY>`**
+
+* To **edit** an API Key name:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/api_key.md]:
```suggestion
* To **edit** an API key name:
```

File: content/en/api/key_management/api_key.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+    **`POST /api/v1/api_key/<API_KEY>`**
+
+* To **delete** an API Key from Datadog:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/api_key.md]:
```suggestion
* To **delete** an API key from Datadog:
```

File: content/en/api/key_management/api_key_code.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+title: API Keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/api_key_code.md]:
```suggestion
title: API keys
```

File: content/en/api/key_management/app_key.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+---
+title: APP Keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
title: APP keys
```

File: content/en/api/key_management/app_key.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+type: apicontent
+order: 16.2
+external_redirect: /api/#app-keys
+---
+
+## APP Keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
## APP keys
```

File: content/en/api/key_management/app_key.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+Manage all your APP Key with the following endpoints:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
Manage all your APP key with the following endpoints:
```

File: content/en/api/key_management/app_key.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+* To **get** all APP keys available for your account:
+
+    **`GET /api/v1/application_key`**
+
+* To **get** a given APP key:
+
+    **`GET /api/v1/application_key/<APP_KEY>`**
+
+* To **edit** an APP Key name:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
* To **edit** an APP key name:
```

File: content/en/api/key_management/app_key.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+    **`POST /api/v1/application_key/<APP_KEY>`**
+
+* To **delete** an APP Key from Datadog:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
* To **delete** an application key from Datadog:
```

File: content/en/api/key_management/app_key_code.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
+---
+title: APP Keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key_code.md]:
```suggestion
title: Application key
```

File: content/en/api/key_management/code_snippets/api_key.sh [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+api_key=<DD_API_KEY>
+app_key=<DD_APP_KEY>
+
+# Get all your current API keys
+curl -X GET \
+  "https://app.datadoghq.com/api/v1/api_key?api_key=${api_key}}&application_key=${app_key}" \
+  -H 'Content-Type: application/json'
+
+# Create a new API Key
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/api_key.sh]:
```suggestion
# Create a new API key
```

File: content/en/api/key_management/code_snippets/app_key.sh [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+api_key=<DD_API_KEY>
+app_key=<DD_APP_KEY>
+
+## Get all your current APP keys
+curl -X GET \
+  "https://app.datadoghq.com/api/v1/application_key?api_key=${api_key}}&application_key=${app_key}" \
+  -H 'Content-Type: application/json'
+
+## Create a new APP Key
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/app_key.sh]:
```suggestion
## Create a new application key
```

File: content/en/api/key_management/code_snippets/result.api_key.sh [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+{
+    "api_key": {
+        "created_by": "john@example.com",
+        "name": "<API_KEY_NAME>",
+        "key": "3111111111111111aaaaaaaaaaaaaaaa",
+        "created": "2019-04-05 09:47:00"
+    }
+}
+
+## Delete an API Key
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/result.api_key.sh]:
```suggestion
## Delete an API key
```

File: content/en/api/key_management/code_snippets/result.app_key.sh [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+
+{
+    "application_key": {
+        "owner": "john@example.com",
+        "hash": "31111111111111111111aaaaaaaaaaaaaaaaaaaa",
+        "name": "<NEW_APP_KEY_NAME>"
+    }
+}
+
+## Delete an API Key
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/result.app_key.sh]:
```suggestion
## Delete an API key
```

File: content/en/api/key_management/key_managment.md [hunk 17]
```diff
@@ -0,0 +1,11 @@
+---
+title: Key Management
+type: apicontent
+order: 16
+external_redirect: /api/#key-management
+---
+
+## Key Management
+
+Manage your Datadog API Key and Application Key. Full list of keys can be seen in the [Datadog API page][1].
```
Comment (ruthnaebeck) [inline content/en/api/key_management/key_managment.md]:
```suggestion
Manage your Datadog API key and application key. The full list of keys can be seen on your [Datadog API page][1].
```

File: content/en/api/key_management/app_key.md [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+type: apicontent
+order: 16.2
+external_redirect: /api/#app-keys
+---
+
+## APP Keys
+
+Manage all your APP Key with the following endpoints:
+
+* To **get** all APP keys available for your account:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
* To **get** all application keys available for your account:
```

File: content/en/api/key_management/app_key.md [hunk 19]
```diff
@@ -0,0 +1,32 @@
...
+
+    **`GET /api/v1/application_key`**
+
+* To **get** a given APP key:
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
* To **get** a given application key:
```

File: content/en/api/key_management/app_key.md [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+
+* To **delete** an APP Key from Datadog:
+
+    **`DELETE /api/v1/application_key/<APP_KEY>`**
+
+##### ARGUMENTS
+
+* **`name`** [*required*]:
+
+    Name of your APP key.
```
Comment (ruthnaebeck) [inline content/en/api/key_management/app_key.md]:
```suggestion
    Name of your application key.
```

File: content/en/api/key_management/code_snippets/app_key.sh [hunk 21]
```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+# Make sure you replace the <DD_API_KEY> and <DD_APP_KEY> key below
+# with the ones for your account
+
+api_key=<DD_API_KEY>
+app_key=<DD_APP_KEY>
+
+## Get all your current APP keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/app_key.sh]:
```suggestion
## Get all your current application keys
```

File: content/en/api/key_management/code_snippets/app_key.sh [hunk 22]
```diff
@@ -0,0 +1,41 @@
...
+## Create a new APP Key
+
+curl -X POST \
+  "https://app.datadoghq.com/api/v1/application_key?api_key=${api_key}&application_key=${app_key}" \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"name": "<APP_KEY_NAME>"
+}'
+
+## Get a given APP KEY name
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/app_key.sh]:
```suggestion
## Get a given application key name
```

File: content/en/api/key_management/code_snippets/app_key.sh [hunk 23]
```diff
@@ -0,0 +1,41 @@
...
+
+curl -X GET \
+  "https://app.datadoghq.com/api/v1/application_key/<APP_KEY_TO_GET>?api_key=${api_key}&application_key=${app_key}" \
+  -H 'Content-Type: application/json'
+
+## Edit a given APP KEY name
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/app_key.sh]:
```suggestion
## Edit a given application key name
```

File: content/en/api/key_management/code_snippets/app_key.sh [hunk 24]
```diff
@@ -0,0 +1,41 @@
...
+
+curl -X PUT \
+  "https://app.datadoghq.com/api/v1/application_key/<APP_KEY_TO_EDIT>?api_key=${api_key}&application_key=${app_key}" \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "name":"<NEW_APP_KEY_NAME>"
+}'
+
+## Delete a given APP key
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/app_key.sh]:
```suggestion
## Delete a given application key
```

File: content/en/api/key_management/code_snippets/result.app_key.sh [hunk 25]
```diff
@@ -0,0 +1,56 @@
+## Get all your APP keys
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/result.app_key.sh]:
```suggestion
## Get all your application keys
```

File: content/en/api/key_management/code_snippets/result.app_key.sh [hunk 26]
```diff
@@ -0,0 +1,56 @@
...
+        },
+        {
+            "owner": "jane@example.com",
+            "hash": "21111111111111111111aaaaaaaaaaaaaaaaaaaa",
+            "name": "<APP_KEY_NAME_2>"
+        }
+    ]
+}
+
+## Create a new APP KEY
```
Comment (ruthnaebeck) [inline content/en/api/key_management/code_snippets/result.app_key.sh]:
```suggestion
## Create a new application key
```

**Comment by ruthnaebeck** [comment]
Just need to resolve the merge conflict

**Comment by l0k0ms** [comment]
@ruthnaebeck merge conflict fixed :)

File: content/en/api/key_management/app_key.md [hunk 27]
```diff
@@ -0,0 +1,32 @@
+---
+title: Application keys
+type: apicontent
+order: 16.2
+external_redirect: /api/#app-keys
+---
+
+## Application keys
```
Comment (apigirl) [inline content/en/api/key_management/app_key.md]: application needs to be capitalized

**Pull request merged**